tests/connector_test.go

```diff
@@ -84,14 +81,14 @@ func TestPullPolicies(t *testing.T) {
 	assert.Equals(t, newTime.After(startTime), true)
 }
 
-func RunStep(t *testing.T, connector deployer.Connector, moduleName string) (*string, any, error) {
+func RunStep(t *testing.T, connector deployer.Connector, moduleName string) (string, any, error) {
 	stepID := "hello-world"
 	input := map[string]any{"name": templatePluginInput}
 
 	plugin, err := connector.Deploy(context.Background(), moduleName)
 
 	if err != nil {
-		return nil, nil, err
+		return "nil", nil, err
```

Comment:
I think the first value should be an empty string.

Reply:
Originally the function returned a string pointer, but that was breaking after the output was changed due to the return from the `atpClient` function being a struct. Is it the right fix to simply make this return a string, or is there a more correct way to still use the pointer in the return as before?